llvm/test/CodeGen/AArch64/GlobalISel/combine-ext.mir

```diff
@@ -6,21 +6,41 @@ body:             |
   bb.1:
   liveins: $x0
     ; CHECK-LABEL: name: test_combine_anyext_trunc
-    ; CHECK: [[COPY:%[0-9]+]]:_(s64) = COPY $x0
```

Comment:
Can you pre-commit the change from CHECK to CHECK-NEXT?

Reply:
Splitting it into 2 commits in the same PR is pointless, as the merge behavior is squash and submit. You can directly push the regen to main

Reply:
Ohh apologies I didn't knew this squash behavior, I am used to bitbucket. I don't have rights to push directly to main, should I create a PR with the update test?

Reply:
Yes

Reply:
Okay, created one: https://github.com/llvm/llvm-project/pull/77756